src/libasr/codegen/asr_to_julia.cpp

```diff
@@ -1440,8 +1440,16 @@ class ASRToJuliaVisitor : public ASR::BaseVisitor<ASRToJuliaVisitor>
     {
         src = "\"";
         std::string s = x.m_s;
-        for (size_t idx=0; idx < s.size(); idx++) {
-            src += s[idx];
+        for (size_t idx = 0; idx < s.size(); idx++) {
+            if (s[idx] == '\n') {
+                src += "\\n";
+            } else if (s[idx] == '\\') {
+                src += "\\\\";
+            } else if (s[idx] == '\"') {
+                src += "\\\"";
+            } else {
+                src += s[idx];
+            }
```

Comment:
I guess this needs to be reverted. Please see https://github.com/lcompilers/lpython/commit/3c53ffb1c217405224ff53f0b3a7a5d306fb07d7.